AutomatedTesting/Gem/PythonTests/scripting/TestSuite_Periodic.py

```diff
@@ -34,12 +34,44 @@ class test_NodePalette_HappyPath_CanSelectNode(EditorBatchedTest):
     class test_EditMenu_Default_UndoRedo(EditorBatchedTest):
         import EditMenu_Default_UndoRedo as test_module
 
+
 @pytest.mark.REQUIRES_gpu
 @pytest.mark.SUITE_periodic
 @pytest.mark.parametrize("launcher_platform", ['windows_editor'])
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
-class TestAutomation(TestAutomationBase):
+class TestAutomationQtPyTests(TestAutomationBase):
+
+    # Enable only -autotest_mode for these tests. Tests cannot run in -BatchMode due to UI interactions
+    global_extra_cmdline_args = []
+
+    def test_ScriptCanvas_ChangingAssets_ComponentStable(self, request, workspace, editor, launcher_platform):
+        from . import ScriptCanvas_ChangingAssets_ComponentStable as test_module
+        self._run_test(request, workspace, editor, test_module)
+
+    def test_VariableManager_ExposeVarsToComponent(self, request, workspace, editor, launcher_platform):
+        from . import VariableManager_ExposeVarsToComponent as test_module
+        self._run_test(request, workspace, editor, test_module)
+
+    def test_VariableManager_UnpinVariableType_Works(self, request, workspace, editor, launcher_platform):
+        from . import VariableManager_UnpinVariableType_Works as test_module
+        self._run_test(request, workspace, editor, test_module)
+
+    def test_ScriptCanvas_TwoComponents_InteractSuccessfully(self, request, workspace, editor, launcher_platform):
+        from . import ScriptCanvas_TwoComponents_InteractSuccessfully as test_module
+        self._run_test(request, workspace, editor, test_module)
+
+    """
+    This test fails in multi test. QCheckbox state change does not trigger table changes like in hydra/editor test run
```

Comment:
Backlog this as a GHI and add the issue to the start of this comment block using the `o3de/o3de#1234 ` pattern.

Reply:
wrote the GHI and referenced it here.
link as well: [#13481](https://github.com/o3de/o3de/issues/13481)